jenkins/run-server-func-tests

```diff
@@ -11,14 +11,55 @@ export PB_SERVER_CONTAINER_NAME=${PB_SERVER_CONTAINER_NAME:-"${PB_POD_NAME}-pben
 SERVER_URL="http://localhost:8080"
 SERVER_API_ENDPOINTS="${SERVER_URL}/api/v1/endpoints"
 
-cleanup_flag=""
+cleanup_flag=0
+keep_flag=0
 exit_status=0
 
-if [[ ${1} == "--cleanup" ]]; then
-    cleanup_flag=1
-elif [[ -n "${1}" ]]; then
-    echo "Unrecognized argument \"${1}\"" >&2
-    exit 2
+function usage {
+	printf "Spin up the necessary containers for the Pbench Server on the local"
+    printf "host and run the server functional tests.\n"
+    printf "\nThe following options are available:\n"
+	printf "\n"
```

Comment:
Some of your indentations are using hard-tabs when I believe we're supposed to be using spaces.

Reply:
Interesting. I thought my `vscode` bash indentation was set to 4 spaces. (And the info bar confirms that.) When I reformatted just now, it changed to spaces, so I have no idea what happened last week. Weird.